### ISSUE TITLE
PICMI detail: run simulation without particle species

### ISSUE THREAD

**Issue by LDAmorim** [ECP-WarpX/WarpX#637]
Hi,
I am trying to test an RZ simulation of a laser propagating in vacuum (i.e. without particles/plasma) with WarpX.
I wish to compare the results with those from the same setup, but ran with code FBPIC (<https://github.com/fbpic/fbpic.git>).

For that reason, I adapted the PICMI (<https://github.com/picmi-standard/picmi.git>) standard script below to remove all the species information.
[picmi_compton.txt](https://github.com/ECP-WarpX/WarpX/files/4074646/picmi_compton.txt)

The script works well with FBPIC that runs it without a problem.

But outputs an error (messages shown below) when producing the input file for WarpX, because it expects to have species in line:
https://github.com/ECP-WarpX/WarpX/blob/eb9de8849b2d5c14f928615def7cb3351bf12744/Python/pywarpx/WarpX.py#L32

This is just a detail.
But my question is (mainly for @dpgrote and @RemiLehe) if there is a PICMI command or line that I should introduce in the script to specify that I wish to model 0 particle species.
I didn't find it in the PICMI documentation (so far).

If there isn't, I could introduce, ahead of the code line written above, something like:
```python
if len(particles.species_names) !=0:
    for pstring in particles.species_names: 
        ...
else:
    particles.nspecies=0
```

What do you think?

Thanks,
Diana

---

### Error messages

```python
python picmi_compton.py 
Traceback (most recent call last):
  File "picmi_compton.py", line 188, in <module>
    sim.write_input_file(file_name='input_script')
  File "/anaconda3/lib/python3.7/site-packages/pywarpx-19.10-py3.7.egg/pywarpx/picmi.py", line 579, in write_input_file
    pywarpx.warpx.write_inputs(file_name, **kw)
  File "/anaconda3/lib/python3.7/site-packages/pywarpx-19.10-py3.7.egg/pywarpx/WarpX.py", line 73, in write_inputs
    argv = self.create_argv_list()
  File "/anaconda3/lib/python3.7/site-packages/pywarpx-19.10-py3.7.egg/pywarpx/WarpX.py", line 32, in create_argv_list
    for pstring in particles.species_names.split(' '):
AttributeError: 'NoneType' object has no attribute 'split'
(base) henrivincenti:NoPlasma diana$ python
tPython 3.7.3 | packaged by conda-forge | (default, Jul  1 2019, 14:38:56) 
[Clang 4.0.1 (tags/RELEASE_401/final)] :: Anaconda, Inc. on darwin
Type "help", "copyright", "credits" or "license" for more information.
```

### Codes versions

- WarpX 0b25b1b68f1b70e965295505d1503a3e57b6c2cc
- AMREX 4cbc0b076a70a299fd4d8b0c51c6c3d6aa34f2ce
- PICSAR eda7243ab99845ade7eaeabb84c971a94cad7645
- FBPIC f253ab8814748ea9a703f4dff2ccf743729c35ef
- Picmi eb23f247c8d741ca52b3f584c5e3d4f46591cdfe

**Comment by dpgrote** [comment]
This issue had already been addressed in PR #593

**Comment by LDAmorim** [comment]
Thank you for the quick fix @dpgrote !
